### PR TITLE
feat(tests): replace firefox with chrome for tests

### PR DESCRIPTION
**Pull request by filipesilva**

### CODE REVIEW

File: addon/ng2/blueprints/ng2/files/karma.conf.js [hunk 1]
```diff
@@ -46,5 +52,11 @@ module.exports = function(config) {
     autoWatch: true,
     browsers: ['Chrome', 'Firefox'],
     singleRun: false
-  });
+  };
+
+  if(process.env.TRAVIS){
+    configuration.browsers = ['Chrome_travis_ci', 'Firefox'];
+  }
+
```
Comment (Brocco) [inline addon/ng2/blueprints/ng2/files/karma.conf.js]: I'd prefer to keep this logic in the CLI's tests, the if condition is already there to force Firefox

Comment (filipesilva) [inline addon/ng2/blueprints/ng2/files/karma.conf.js]: Ah I was wondering where that was! Cheers!

**Comment by Brocco** [comment]
Merged

**Comment by zsfarkas** [comment]
Hi, 

is there any easy way to use Firefox for tests? I mean by setting an argument like 

`ng test --firefox`

or something like that.

Thank you.

Zsolt

**Comment by filipesilva** [comment]
@zsfarkas you can either do `ng test --browsers=firefox` or just edit your karma config.

**Comment by Brocco** [comment]
@zsfarkas also note that you will need to include a dependency to karma-firefox-launcher...

```bash
npm install karma-firefox-launcher --save-dev
```

**Comment by zsfarkas** [comment]
Thank you. It works.

**Pull request closed without merging**